app/controllers/profile/orders_controller.rb

```diff
@@ -2,4 +2,22 @@ class Profile::OrdersController < Profile::BaseController
   def index
     @user = current_user
   end
+
+  def show
+    @order = Order.find(params[:id])
+  end
+
+  def update
```

Comment:
This feels more like a destroy action? Since we're getting rid of the order, even if we're not deleting it from the database.

Reply:
Agree with Alex, good work on getting it updated.

Reply:
good catch, updated